card/mod/history/lib/card/action.rb

```diff
@@ -25,6 +25,7 @@ class Action < ApplicationRecord
     extend Card::Action::Admin
 
     belongs_to :act, foreign_key: :card_act_id, inverse_of: :ar_actions
+    belongs_to :ar_card, foreign_key: :card_id, inverse_of: :actions, class_name: "Card"
```

Comment:
Neither of likes this nomenclature.

I would propose the following (but I'm not sure if it works):
1. for acts, actions, changes, and references, we use the full name (eg `card_acts`) to refer to the active record relations and the short name (eg `acts`) to refer to the caching version.
2. for cards, we can't do this, but I think it's pretty important that, eg, `@action.card` use the cache.  if it's possible to use both `belongs_to :card` and `def card` harmoniously (eg, so that we can do `joins(card_actions: :card) above`), that would be great.  But I'm not sure it will work.  If not `ar_card`, what is another option?  `card_model`?  `card_relation`?